grabba_grabba_hey/sentinel_downloader.py

```diff
@@ -171,69 +173,91 @@ def parse_xml(xml):
     # for x in img.getchildren():
 
 
-def download_sentinel(location, input_start_date, input_sensor, output_dir,
-                      input_end_date=None, username="guest", password="guest"):
-    input_sensor = input_sensor.upper()
-    sensor_list = ["S1", "S2"]
-    if not input_sensor in sensor_list:
-        raise ValueError("Sensor can only be S1 or S2. You provided %s"
-                         % input_sensor)
+def download_sentinel(location, input_start_date, 
```

Comment:
Good, can you just name this function as download_sentinel3 or something like that? The idea is that we expose this as the main API. Also, if you fancy some py.test ;-)